index/segment/mem/terms_dict_test.go

```diff
@@ -0,0 +1,170 @@
+package mem
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/m3db/m3ninx/index/segment"
+
+	"github.com/m3db/m3ninx/doc"
+)
+
+var (
+	benchDocs []doc.Document
+)
+
+func init() {
+	var err error
+	if benchDocs, err = readDocuments("../../../testdata/node_exporter.json", 2000); err != nil {
```

Comment:
probably better to use the absolute path 

```
path := os.ExpandEnv("${GOPATH}/github.com/m3db/m3ninx/testdata/node_exporter.json"
... 
```

Reply:
The Go toolchain guarantees that [test binaries will be executed with the working directory set to the source directory of the package the test is in](https://dave.cheney.net/2016/05/10/test-fixtures-in-go) so I believe the convention is to actually [use relative paths from that directory](https://github.com/golang/go/blob/964639cc338db650ccadeafb7424bc8ebb2c0f6c/src/compress/flate/reader_test.go#L25).

Reply:
My main concern was if you moved this snippet of code around (up/down) a package, it'd be more robust to copy/paste if use the absolute path. Either way is fine, up to you.